content/english/python-discord-translate/Setup/code-prerequisites/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Code Prerequisites"
+draft: false
+weight: 2
+---
+
+Nuvi can’t invite a translator bot yet—there’s no code environment set up! Let’s get your computer ready. You’ll install Python, set up a virtual environment (a safe “bubble” for your project’s packages), install VS Code (your code editor), and create the starter files. Once this foundation is ready, you’ll start writing the bot that helps everyone understand each other.
+
+{{< alert theme="info" >}}
+<img src="../../media/NF_mascot.jpg" alt="Nuvi mascot" width="90" style="float:right;margin:0 0 8px 12px;" />
+<strong>Nuvi says:</strong> Pick Python 3.10 so the translator parts install smoothly—future upgrades are easier once things work now!
+{{< /alert >}}
+
+## 1. Install Python
+Install **Python 3.10** (a 3.10.x version, like 3.10.14) from the official website.
+- Windows & macOS: https://www.python.org/downloads/
+- (Some Chromebooks may need Linux mode; ask an adult or teacher.)
+
+Why 3.10? Some of the libraries we will use for Discord and translation work best (or only) with Python 3.10 right now. Using a newer version (like 3.11 or 3.12) may cause install errors.
+
+You should see something like `Python 3.10.x`.
+
+## 2. Install VS Code (and the Python tools)
+
+VS Code is a free, student‑friendly code editor where you'll write and run your bot; it gives smart autocompletion, error hints, and an easy way to manage Python.
+
+Download VS Code from: https://code.visualstudio.com/
+
+After VS Code installs, add the Python tooling:
+1. Open VS Code and go to the Extensions view (Click on the extensions icon in the Activity Bar on the side or press `Ctrl+Shift+X`).
+![Extensions view icon](../../media/extensions.png)
+2. Search for `Python` (Publisher: Microsoft) and click Install. This will install some helpful tools we'll be using throughout the workshop. ![Python VSCode extension page](../../media/python-extension.png)
+3. Install the `Python Environment Manager` experience if prompted – it is bundled in the Python extension recent versions.
+4. Reload VS Code if it asks you to.
+
+You should now see the Python extension symbol on your sidebar.
+![Python extension symbol](../../media/python-symbol.png)
+
+## 3. Create a Project Folder
+Choose a location you can find easily, like your Documents or Desktop.
+Example folder name: `discord-translator-bot`.
+Open the folder in VS Code (File → Open Folder).
+
+## 4. Create a Virtual Environment
+A virtual environment keeps your project separate from other projects and makes it easier to organize all of the Python files on your computer. The VS Code Python extension can create and select it for you.
+
+{{< alert theme="info" >}}
+<img src="../../media/NF_mascot.jpg" alt="Nuvi mascot" width="70" style="float:right;margin:0 0 6px 10px;" />
```

Comment:
Can you specify an exact px value here instead of just 70? Just so that each browser can interpret this consistently.